### PR TITLE
initial commit of windows log collection script

### PR DESCRIPTION
**Pull request by rosskirkpat**
Adding a new log collection script for Windows

### CODE REVIEW

File: collection/rancher/v2.x/windows-log-collector/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# rancher-windows-log-collector
+Windows log collector for Rancher Windows Worker Nodes
+
+
+## How To Run the Script
+
+```ps1
+Set-ExecutionPolicy Bypass
+Start-BitsTransfer https://raw.githubusercontent.com/rosskirkpat/rancher-windows-log-collector/master/log-collect-beta.ps1
```
Comment (dnoland1) [inline collection/rancher/v2.x/windows-log-collector/README.md:9]: Need to fix this URL to point to support-tools repo

File: collection/rancher/v2.x/windows-log-collector/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# rancher-windows-log-collector
+Windows log collector for Rancher Windows Worker Nodes
+
+
+## How To Run the Script
```
Comment (dnoland1) [inline collection/rancher/v2.x/windows-log-collector/README.md:5]: For people who have little/no windows experience, might be worth mentioned that they need to open a PS or CMD or whatever type of window to run this in.

File: collection/rancher/v2.x/windows-log-collector/README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+```ps1
+Set-ExecutionPolicy Bypass
+Start-BitsTransfer https://raw.githubusercontent.com/rosskirkpat/rancher-windows-log-collector/master/log-collect-beta.ps1
+.\log-collect-beta.ps1
```
Comment (dnoland1) [inline collection/rancher/v2.x/windows-log-collector/README.md:13]: Needs to be corrected to `.\win-log-collect.ps1`

File: collection/rancher/v2.x/windows-log-collector/README.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+```
+
+### Expected output
+#### Note: The `Unable to Collect Windows Firewall information` error is expected if it there are not Domain specific firewall rules pushed out by the customer
+
+```ps1
+ .\log-collect-beta.ps1
```
Comment (dnoland1) [inline collection/rancher/v2.x/windows-log-collector/README.md:20]: Needs to be corrected to `.\win-log-collect.ps1`

Comment (rosskirkpat) [reply collection/rancher/v2.x/windows-log-collector/README.md:20]: fixed this

File: collection/rancher/v2.x/windows-log-collector/README.md (hunk 2 above)
Comment (rosskirkpat) [reply collection/rancher/v2.x/windows-log-collector/README.md:5]: I updated the instructions, mind giving them a +1? @dnoland1

File: collection/rancher/v2.x/windows-log-collector/README.md (hunk 1 above)
Comment (rosskirkpat) [reply collection/rancher/v2.x/windows-log-collector/README.md]: updated the URL to point to the `windows-log-collect` branch in the `support-tools` repo

File: collection/rancher/v2.x/windows-log-collector/README.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+OK
+Collecting network Information
+OK
+Collecting group policy information
+Get-GPOReport is not a valid cmdlet
+Collecting proxy information
+OK
+Archiving Rancher log collection script data
+OK
+Done. Your log bundle is located in  C:\rancher\rancher_EC2AMAZ-ENEJ0H8_20200602T1704290242Z
```
Comment (dnoland1) [inline collection/rancher/v2.x/windows-log-collector/README.md:76]: Can this path be fixed?

Comment (rosskirkpat) [reply collection/rancher/v2.x/windows-log-collector/README.md]: path fixed in README.md and in the script itself

**Pull request merged**